dialects/duckdb.yaml

```diff
@@ -29,9 +29,15 @@ scalar_functions:
   - name: modulus
     local_name: "%"
     infix: True
-    required_options:
```

Comment:
As per substrait documentation, no options are required for the modulus function.

Reply:
since division occurs as part of modulus, we'll probably need to update the substrait documentation to include more options.  I can create a PR for that.

Reply:
made the change, thanks!